commands/add/__tests__/add-command.test.js

```diff
@@ -44,6 +44,15 @@ describe("AddCommand", () => {
     await expect(command).rejects.toThrow(/Requested package has no version:/);
   });
 
+  it("should throw when using pnpm", async () => {
+    const testDir = await initFixture("pnpm");
+    const command = lernaAdd(testDir)("@test/package-1");
+
+    await expect(command).rejects.toThrow(
+      "The 'add' command is not supported when using `pnpm` workspaces. Use `pnpm` directly to add dependencies to packages: https://pnpm.io/cli/add."
```

Comment:
I think it's cleaner to not have the period after the URL (and maybe it would mess with terminals trying to make it clickable?)

```suggestion
      "The 'add' command is not supported when using `pnpm` workspaces. Use `pnpm` directly to add dependencies to packages: https://pnpm.io/cli/add"
```